docs/about/contributing.rst

```diff
@@ -9,22 +9,24 @@ there is an easy way to improve the tool for both of us.
 
 Also please create a GitHub issue if anything doesn't appear to be
 working right. If you get an unexpected error, please include the full
-stack trace that you get with ``^E``.
+stack trace that you get with ``Ctrl-e``.
 
 Branch structure
-~~~~~~~~~~~~~~~~
-
-Visidata has two main branches: \*
-`stable <https://github.com/saulpw/visidata/tree/stable>`__ has the last
-known good version of visidata. \*
-`develop <https://github.com/saulpw/visidata/tree/develop>`__ has the
-most up-to-date version of visidata (which will eventually be merged to
-stable).
-
-If you wish to contribute, please fork from
-`develop <https://github.com/saulpw/visidata/tree/develop>`__ and submit
-a `pull request <https://github.com/saulpw/visidata/pulls>`__ against
-it.
-
-A developer's guide can be found
-`here <http://visidata.readthedocs.io/en/stable/dev-guide>`__.
+----------------
+
+Visidata has two main branches:
+
+* `stable <https://github.com/saulpw/visidata/tree/stable>`__ has the last
+  known good version of visidata.
+
+* `develop <https://github.com/saulpw/visidata/tree/develop>`__ has the
+  most up-to-date version of visidata (which will eventually be merged to
+  stable).
+
+If you wish to contribute, please fork from `develop
+<https://github.com/saulpw/visidata/tree/develop>`__ and submit a `pull request
+<https://github.com/saulpw/visidata/pulls>`__ against it.
+
+A developer's guide can be found `here
+<http://visidata.readthedocs.io/en/stable/dev-guide>`__.
```

Comment:
Oh, that link has been broken.
It should be: https://visidata.readthedocs.io/en/stable/dev-guide.html